src/bindings/python/src/pyopenvino/graph/node_output.hpp

```diff
@@ -41,6 +41,27 @@ void regclass_graph_Output(py::module m, std::string typestring)
                 get_index : int
                     Index value as integer.
                )");
+    output.def("get_any_name",
```

Comment:
from my perspective it also would be nice to add properties

Reply:
done